protocol/xmrmaker/instance_test.go

```diff
@@ -157,6 +157,10 @@ func TestInstance_createOngoingSwap(t *testing.T) {
 	rate := coins.ToExchangeRate(apd.New(1, 0)) // 100% relayer commission
 	offer := types.NewOffer(coins.ProvidesXMR, one, one, rate, types.EthAssetETH)
 
+	offerDB.EXPECT().PutOffer(offer).Return(nil)
+	_, err := inst.offerManager.AddOffer(offer, "", nil)
+	require.NoError(t, err)
```

Comment:
I'm still confused as to how this test previously worked. I don't see anyone adding the offer to the database or the cache that the offer manager is using.